samples/powerstore_v260_k8s_123.yaml

```diff
@@ -5,16 +5,17 @@ metadata:
   namespace: test-powerstore
 spec:
   driver:
-    # Config version for CSI PowerStore v2.3.0 driver
-    configVersion: v2.3.0
+    # Config version for CSI PowerStore v2.6.0 driver
+    configVersion: v2.6.0
     # Controller count
     replicas: 2
     dnsPolicy: ClusterFirstWithHostNet
     forceUpdate: false
     fsGroupPolicy: ReadWriteOnceWithFSType
+    storageCapacity: true
```

Comment:
Storage Capacity Tracking is not supported in k8s 1.23

Reply:
Addressed.
Thank You.!